package.json

```diff
@@ -43,7 +43,7 @@
     "vl-ui-core": "^3.0.1",
     "vl-ui-grid": "^1.1.0",
     "vl-ui-titles": "^1.0.13",
-    "vl-ui-typography": "^1.0.4"
+    "vl-ui-typography": "milieuinfo/webcomponent-vl-ui-typography#VOORTOETS-3050"
```

Comment:
Zou best nog aangepast worden voor een release, denk ik. Anders gaat een npm install van een release van deze module falen van zodra de branch weg is.